README.md

```diff
@@ -167,6 +167,55 @@ carthage archive YourFrameworkName
 
 Draft Releases will be automatically ignored, even if they correspond to the desired tag.
 
+#### Use travis-ci to upload your tagged prebuild frameworks
+
+It is possible to use travis-ci in order to build and upload your tagged releases.
+
+1. [Install travis CLI](https://github.com/travis-ci/travis.rb#installation) with `gem install travis`
+1. [Setup](https://docs.travis-ci.com/user/getting-started/) travis-ci for your repository (Steps 1 and 2)
+1. Create `.travis.yml` file at the root of your repository based on that template. Set `XCODE_SCHEME` as well as `FRAMEWORK_NAME` to your correct values
+
+	```
+	language: objective-c
+	osx_image: xcode7.1
+	env:
+		global: 
+		- XCODE_SCHEME=<THIS_IS_A_PLACEHOLDER_REPLACE_ME>
```

Comment:
Actually we don't need it for building the framework. If we want to have multiple options, let's put some gists there for the non travis aficionados.